modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java

```diff
@@ -6075,9 +6075,13 @@ protected String addImageFileEntries(JournalArticle article, String value)
 					folder.getGroupId(), folder.getFolderId(),
 					tempFileEntry.getFileName(), false);
 
+				// We need to use the UUID as the external reference code to
+				// keep track of the temporary file entry when autosaving a Web
+				// Content. See LPD-52357.
+
```

Comment:
Hey! This was just a random example! :-D But I don't think it is correct, isn't the real reason something like:

>  We need to use the UUID as the external reference code of the attachment file to keep track of its relationship with the original temporary file during autosaving. The frontend will keep track of the initial UUID, and it will use that to indicate that the document needs to be updated when autosave is triggered.

But don't copy it like verbatim. You know the issue much better than me, so feel free to update it to reflect the actual reason.

Reply:
Please do not make me think!!!!

<img src="https://media4.giphy.com/media/v1.Y2lkPWJkM2VhNTdlbDVkMHM1OHR4c3EweTg4OHEwbHJ0dmdmMzNxMXZuZXc2OTAxeDJ2NiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/hiEVPOfy04kC70RU7R/giphy.gif"/>

Reply:
```
				// The UUID for the temporary file is persisted on page but the
				// file will be deleted on autosaving, and we need to keep track
				// of the that UUID. To do keep track of it we will use the
				// temporary file UUID as ERC for the final file. See LPD-52357.
```

It's good enough ?

@robertoDiaz